hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/actionhistory/ActionStatusIconMapper.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.ui.management.actionhistory;
+
+import java.util.Map;
+
+import org.eclipse.hawkbit.repository.model.Action;
+import org.eclipse.hawkbit.repository.model.ActionStatus;
+
+import com.google.common.collect.Maps;
+import com.vaadin.server.FontAwesome;
+
+/**
+ * Mapping utility for {@link ActionStatus} to icon in action history table.
+ *
+ */
+public final class ActionStatusIconMapper {
+    static final String STATUS_ICON_GREEN = "statusIconGreen";
```

Comment:
There are constants for these styles `org.eclipse.hawkbit.ui.utils.SPUIDefinitions`